testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/LinkedAccountsTest.java

```diff
@@ -171,6 +192,36 @@ public void unlinkAccountTest() {
         assertEquals(0, testUserResource().getFederatedIdentity().size());
     }
 
+    @Test
+    public void unlinkWebAuthnTest() {
+        signingInPage.navigateTo();
+
+        String passwordId = testUserResource().credentials().get(0).getId();
+        testUserResource().removeCredential(passwordId);
+
+        SigningInPage.CredentialType webAuthnPwdlessCredentialType = signingInPage.getCredentialType(WebAuthnCredentialModel.TYPE_PASSWORDLESS);
+        webAuthnPwdlessCredentialType.clickSetUpLink();
+
+        webAuthnRegisterPage.confirmAIA();
+        webAuthnRegisterPage.registerWebAuthnCredential("WebAuthn is convenient");
+        waitForPageToLoad();
```

Comment:
I believe this could've been done just by using admin REST endpoints (UI is tested in another test) but that's ok since you already have this.